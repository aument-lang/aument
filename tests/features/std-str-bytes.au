let abc = str::bytes('ABC');
print abc[0];
print abc[1];
print abc[2];