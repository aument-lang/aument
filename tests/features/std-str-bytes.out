int;65
int;66
int;67